client/css/bootstrap.css

```diff
@@ -1022,6 +1022,127 @@ tbody.collapse.in {
   -o-transition: height 0.35s ease;
   transition: height 0.35s ease;
 }
+.dropdown {
```

Comment:
I've actually copied the styling entirely from bootstrap - This particular css exists in extended bootstrap with the dropdown included. That's why I appended this to bootstrap.css and not generic css. Plus I think the div that is added has the class `.dropdown` by default (customizable but default is .`dropdown` for compatibility with extended bootstrap.css)